pages/linux/sic.md

```diff
@@ -0,0 +1,29 @@
+# sic
+
+> simple irc client.
+> Part of suckless.org tools.
+> More information: <https://tools.suckless.org/sic/>.
+
+- Connect to host (default: irc.oftc.net) with nickname (default: $USER):
```

Comment:
You don't specify the host here to connect to, so I'm not sure the description here makes sense. What about this?
```suggestion
- Connect to irc.oftc.net with a given nickname (default: `$USER`):
```

Reply:
Devil is in the details, the explain part mentions host and the default.
Compromise: how about `connect to default (irc.ofct.net) irc host` ? 
Though I like my version better.

Reply:
How about instead `Connect to host with a given nickname:` and instead of `nickname` in the example itself use `{{$USER}}` and show add `-h {{irc.ofct.net}}`?

Reply:
See, this is what I'm thinking of: https://github.com/tldr-pages/tldr/pull/5694/files#r607929529